src/Domain/FunctionsOfTime/FunctionOfTime.hpp

```diff
@@ -37,6 +37,21 @@ class FunctionOfTime : public PUP::able {
       noexcept = 0;
 
   WRAPPED_PUPable_abstract(FunctionOfTime);  // NOLINT
+
+  // Functions in the DataBox must be streamable, because operator<<() is
```

Comment:
I should amend to say "An object added as a `SimpleTag` in the DataBox must either be streamable itself or have support for streaming provided by `StdHelpers.hpp`. Otherwise, an error message in `merge_simple_tag_value` will fail to compile"

Reply:
Note to self: when mutating instead of adding to databox, maybe don't need this. But if you do need it, make the stream operator virtual and print out entire internal state

Reply:
Now that my action is a mutating action, this is no longer an issue, so I no longer am going to try to make FunctionOfTime streamable